examples/gno.land/r/sys/params/params.gno

```diff
@@ -19,36 +19,106 @@ package params
 
 import (
 	"std"
+	prms "sys/params"
 
 	"gno.land/p/demo/dao"
 	"gno.land/r/gov/dao/bridge"
+
+	// loads the latest DAO implementation in the bridge.
+	_ "gno.land/r/gov/dao/init"
 )
 
+// XXX What is the point of this realm local parameter?
```

Comment:
It used to be for when important keys were under r/sys/params.important_keys; we can remove it now.